packages/server/src/api/controllers/table/utils.ts

```diff
@@ -88,6 +89,24 @@ export async function checkForColumnUpdates(
     // Update views
     await checkForViewUpdates(updatedTable, deletedColumns, columnRename)
   }
+
+  for (const attachmentColumn of Object.values(updatedTable.schema).filter(
+    column =>
+      column.type === FieldType.ATTACHMENT &&
+      column.subtype !== oldTable?.schema[column.name]?.subtype
+  )) {
```

Comment:
This is a little bit gnarly, might be a bit easier to read if it's split out into its own variable:

```typescript
const changedAttachmentColumns = Object.values(updatedTable.schema).filter(
  column =>
    column.type === FieldType.ATTACHMENT &&
    column.subtype !== oldTable?.schema[column.name]?.subtype
)
for (const attachmentColumn if changedAttachmentColumns) {
  // ...
```